cli/cmd/lql_update.go

```diff
@@ -0,0 +1,66 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// lqlUpdateCmd represents the lql update command
+	lqlUpdateCmd = &cobra.Command{
+		Use:   "update",
+		Short: "update an LQL query",
+		Long:  `Update an LQL query.`,
+		Args:  cobra.NoArgs,
```

Comment:
?? This doesn't seem right, don't we need at least an ID of the query to update?

Am I missing something 🤔

Reply:
The update API is slightly wonky for the following reasons:

1. The API will take the LQL_ID from the QUERY_TEXT
2. You can specify an LQL_ID in the JSON POST body
3. You can specify an LQL_ID argument in the query param

- So if you're using just LQL syntax, or a JSON with just QUERY_TEXT then LQL ID will be inferred.
- If you're using JSON you can specify LQL_ID in the JSON
- If you're using JSON and don't specify LQL_ID in the JSON perhaps you want to specify it as a CLI argument?

I think we should have a separate ticket to sort this all out.

Reply:
ALLY-442